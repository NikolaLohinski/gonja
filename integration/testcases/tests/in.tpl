o/foo: {{ "o" is in "foo" }}
foo/foo: {{ "foo" is in "foo" }}
b/foo: {{ "b" is in "foo" }}
1/(1,2): {{ 1 is in (1, 2) }}
3/(1,2): {{ 3 is in (1, 2) }}
1/[1,2]: {{ 1 is in [1, 2] }}
3/[1,2]: {{ 3 is in [1, 2] }}
foo/{"foo": 1}: {{ "foo" is in {"foo": 1}}}
baz/{"bar": 1}: {{ "baz" is in {"bar": 1}}}
