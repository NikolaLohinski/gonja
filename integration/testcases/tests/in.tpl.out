o/foo: True
foo/foo: True
b/foo: False
1/(1,2): True
3/(1,2): False
1/[1,2]: True
3/[1,2]: False
foo/{"foo": 1}: True
baz/{"bar": 1}: False
